### DISCUSSION TITLE
WMS Documentation

### DISCUSSION THREAD

**Discussion by BMarques** [Unidata/tds#546] [General]
When reading the documentation for the WMS in Thredds, it mentions something like:
"A detailed description of the wmsConfig.xml file can be found at the MyOcean “Detailed WMS Configuration” page."

I don't know where to find this Detailed WMS Configuration, tried googling it, but didn't return anything.

**Comment by lesserwhirls** [comment]
Greetings @BMarques!

The information you are looking for is located at https://docs.unidata.ucar.edu/tds/current/adminguide/customizing_wms.html. We have been in the process of modifying our documentation to separate out end user documentation from server administrator documentation, and the admin documentation is not quite ready for full use. We try to mirror the changes to both so that information is at least findable, but this is one of the cases where we didn't quite mirror correctly. I'll get that fixed shortly, but in the meantime, you can visit the link above for the info.

Cheers!

Sean

**Comment by BMarques** [comment]
Thank you! I wasn't really aware of the adminguide, only could find the userguide, but i'll keep in mind it's still WIP, it's good info nevertheless

**Comment by lesserwhirls** [comment]
We don't quite have the admin guide in shape for release, so it's not officially linked anywhere at the moment. That said, I just made a PR to update the userguide to include the detailed config so that it is at least accessible 😄
